### PR TITLE
Add support for invoking localytics events API

### PR DESCRIPTION
**Pull request by matthewhively**
Add support for invoking localytics beta events API.

From our localytics fork

### CODE REVIEW

File: lib/localytics/event.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    #                           Values must be formatted as strings.
+    # @param ltv_change = Optional int representing incremental change in user's lifetime value.
+    #                     Must be integer, e.g. use 299 for $2.99.
+    def self.send(app_id, customer_id, event_name, event_attributes=nil, ltv_change=nil, api_key=nil, api_secret=nil)
+      raise Error.new('No APP id provided') unless app_id ||= self.app_id
+      raise Error.new('No customer_id provided') if customer_id.nil?
+      raise Error.new('No event_name provided') if event_name.nil? || event_name.empty?
+
+      params = {
+        schema_url:  "https://localytics-files.s3.amazonaws.com/schemas/eventsApi/v0.json",
```
Comment (marcoschicote) [inline lib/localytics/event.rb]: Is this the latest schema definition available? There seems to be a `v1` https://docs.localytics.com/dev/events-api.html#events-api-testing

File: lib/localytics/event.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      }
+      params[:attributes] = event_attributes if event_attributes && !event_attributes.empty?
+      params[:ltv_change] = ltv_change if ltv_change
+
+      Localytics.request api_base, :post, url, api_key, api_secret, params
+    end
+
+    private
+    def self.api_base
+      "https://analytics.localytics.com/events/v0"
```
Comment (marcoschicote) [inline lib/localytics/event.rb]: Same comment applies: https://docs.localytics.com/dev/events-api.html#events-api-events-api-transactions 
> The most recent [api_version] is v1. This version may change in the future releases of Events API. The api version v0 that was used during the beta period is deprecated and its support will be removed in future versions.

Comment (sidcpatel) [reply lib/localytics/event.rb]: Thanks for the feedback. We will update the API to latest.

File: lib/localytics/event.rb (hunk 1 above)
Comment (sidcpatel) [reply lib/localytics/event.rb]: Thanks for the feedback. We will update the API to latest.

Comment (matthewhively) [reply lib/localytics/event.rb]: @marcoschicote The API endpoint has been updated.

**Comment by matthewhively** [comment]
API endpoint updated.

**Pull request merged**